snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/screen/ScreenViewAutotrackingTest.kt

```diff
@@ -69,9 +61,10 @@ class ScreenViewAutotrackingTest {
         NotificationCenter.postNotification("SnowplowScreenView", mapOf(
             "event" to ScreenView(name = "Screen2").activityClassName("Screen2")
         ))
-        Thread.sleep(200)
+        Thread.sleep(2000)
 
-        Assert.assertEquals(2, numberOfScreenViews)
+        val numberOfScreenViews = eventSink.trackedEvents.filter { it.schema == TrackerConstants.SCHEMA_SCREEN_VIEW }
```

Comment:
nit: strange name for a list, I was expecting an int

Reply:
Good catch, fixed now.